fairscale/nn/pipe/async_pipe.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from .multiprocess_pipe import MultiProcessPipe
+from .types import PipelineStyle
+
+
+class AsyncPipe(MultiProcessPipe):
```

Comment:
Are you planning to separate out Async related stuff from pipeline.py as well in this PR?

Reply:
Planning to do that in a followup. Too much to take on in one PR.